cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -89,6 +85,7 @@ virtual machine, the operation fails and the message `Name is already used by an
 As a workaround, create the template from the command line.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1717930[*BZ#1717930*])
 
-* When you create your virtual machine using the wizard, your virtual machine’s
-storage medium must support Read-Write-Many (RWM) PVCs.
+* Read-Write-Many (RWM) PVCs are the only supported storage for live migration,
```

Comment:
In other parts of the documentation we refer to this as `ReadWriteMany`. I'd also change the sentence from `supported storage` to say `supported storage access mode`. For instance:

> * ReadWriteMany (RWX) is the only supported storage access mode for live...